tests/integration/test_manipulate_statistics/test.py

```diff
@@ -56,26 +56,26 @@ def run_test_single_node(started_cluster):
     check_stat_file_on_disk(node1, "test_stat", "all_1_1_0", "b", True)
     check_stat_file_on_disk(node1, "test_stat", "all_1_1_0", "c", True)
 
-    node1.query("ALTER TABLE test_stat DROP STATISTIC a type tdigest")
+    node1.query("ALTER TABLE test_stat DROP STATISTICS a")
```

Comment:
I stumbled upon this while reviewing the tests.

There is a difference between `DROP STATISTICS` (aka. "Removes statistic description from tables metadata and deletes statistic files from disk.", [docs](https://clickhouse.com/docs/en/sql-reference/statements/alter/statistic) and `CLEAR STATISTICS` ("Deletes statistic files from disk.").

To the uninitiated reader, it is not clear how how they differ, in particular which state the table is in after CLEAR. What is missing in the docs is an explanation that statistics are created per-part and lazily, i.e. while statistics are _defined_ per column, individual (column) parts can or cannot have statistics files. This is similar to skipping indexes and projections which are also created lazily. I'd say for
- 'DROP STATISTICS': `Removes statistics from the metadata of the specified columns and deletes all statistics objects in all parts for the specified columns`
- `CLEAR STATISTICS`: `Deletes all statistics objects in all parts for the specified columns. Statistics objects can be rebuild using ALTER TABLE MATERIALIZE STATISTICS`.